internal/commands/open.go

```diff
@@ -29,6 +29,16 @@ func OpenStartIncidentDialog(client bot.Client, triggerID string) error {
 		})
 	}
 
+	incidentTitle := &slack.TextInputElement{
+		DialogInput: slack.DialogInput{
+			Label:       "Incident Title",
+			Name:        "incident_title",
+			Type:        "text",
+			Placeholder: "my-incident-title",
```

Comment:
I think this placeholder could cause confusion because looks like a channel name, maybe could be
```suggestion
			Placeholder: "My Incident Title",
```